maat-court-data-api/src/main/java/gov/uk/courtdata/unlink/validator/UserIdValidator.java

```diff
@@ -16,7 +16,7 @@
     @Override
     public Optional<Void> validate(String userId) throws ValidationException {
 
-        if (!userId.isEmpty() && userId.length()>0) {
+        if (userId != null) {
```

Comment:
Don't we need to check if the user-id is an empty string or not?

Reply:
Good Point. Fixed it now!